tests/test_app.py

```diff
@@ -220,13 +228,179 @@ def test_xarray_part(
     xarray_query_params,
     arctic_bounds: Tuple[float, float, float, float],
 ) -> None:
-    """Test /part endpoint for xarray backend"""
+    """Test /bbox endpoint for xarray backend"""
     response = app.get(
         f"/bbox/{','.join(str(coord) for coord in arctic_bounds)}.tif",
         params={
             **xarray_query_params,
-            "format": "tif",
         },
     )
     assert response.status_code == 200
     assert response.headers["content-type"] == "image/tiff; application=geotiff"
+
+    size = (10, 10)
+    with pytest.warns():
+        response = app.get(
+            f"/bbox/{','.join(str(coord) for coord in arctic_bounds)}/{'x'.join(str(x) for x in size)}.png",
+            params={
+                **xarray_query_params,
+            },
+        )
+    assert response.status_code == 200
+    assert response.headers["content-type"] == "image/png"
+
+    image_data = io.BytesIO(response.content)
+    Image.open(image_data)
+
+    # Check dimensions
+    # assert image.size == size, f"Expected image size {size}, but got {image.size}"
```

Comment:
👀

Reply:
I added this test then realized that the `XarrayReader` in rio-tiler did not support height/width arguments yet, but I think you added those in the last release so I'll fix this test when we handle the upgrade to rio-tiler and titiler.